docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -162,6 +256,26 @@ networkAnnotation:
   annotation2: 'bar'
 ```
 
+- Overrides to add custom resource allocation for YB master & tserver pods
```

Comment:
Let's comment that these override any instance types selected in the universe creation flow.